.github/workflows/new-milestone-and-issue-in-other-repo.yml

```diff
@@ -12,6 +12,7 @@ on:
 
 permissions:
   contents: write
+  issues: write
```

Comment:
Reason for
```
  issues: write
```

is insuffcient permissions [as shown in this failing job](https://github.com/stoyanK7/checkstyle/actions/runs/4015533499/jobs/6897372290#step:3:38).

According to the [docs](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#issues), we need to grant `write` access to `issues` to call `PATCH /repos/{owner}/{repo}/milestones/{milestone_number}`.